e2eAIOK/DeNas/search.py

```diff
@@ -9,13 +9,16 @@
 from e2eAIOK.DeNas.cv.supernet_transformer import Vision_TransformerSuper
 from e2eAIOK.DeNas.nlp.supernet_bert import SuperBertModel, BertConfig
 from e2eAIOK.DeNas.nlp.utils import generate_search_space
+from e2eAIOK.DeNas.thirdparty.supernet_hf import SuperHFModel
 from e2eAIOK.DeNas.asr.supernet_asr import TransformerASRSuper
 from e2eAIOK.DeNas.search.SearchEngineFactory import SearchEngineFactory
 from e2eAIOK.DeNas.search.utils import Timer, parse_config
+from transformers import logging
+logging.set_verbosity_error()
 
 def parse_args(args):
     parser = argparse.ArgumentParser('DE-NAS')
-    parser.add_argument('--domain', type=str, default=None, choices=['cnn', 'vit', 'bert', 'asr'], help='DE-NAS search domain')
+    parser.add_argument('--domain', type=str, default=None, choices=['cnn', 'vit', 'bert', 'asr', 'thirdparty'], help='DE-NAS search domain')
```

Comment:
replace 'thirdparty' with 'huggingface' since we not actually support "thirdparty"